assets/css/admin/style.scss

```diff
@@ -93,12 +93,33 @@ body.post-type-download #screen-meta {
 	padding: 0 20px;
 }
 
+#edd-header-branding {
+	vertical-align: middle;
+}
+
 #edd-header img {
-	display: block;
 	max-width: 300px;
 	margin: 0 0 0 20px;
 }
 
+.edd-header-page-title-wrap {
+	margin-top: 7px;
+	font-size: 22px;
+	line-height: 28px;
+	vertical-align: top;
+	display: inline;
+	margin-right: auto;
+	padding-left: 7px;
+}
+
+.edd-header-seperator {
```

Comment:
If we are going to make this change can the rule and element be updated to use `separator` please (spelling)?